purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1821,6 +1856,10 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
          */
         @JvmOverloads
         @JvmStatic
+        @Deprecated(
+            message = "Parameters have been replaced with a Builder",
```

Comment:
do we have a ticket somewhere to remember to increase these to "error"?

Reply:
I am not increasing these to Error since they actually have a conversion path already in place. They will work fine but people will be encouraged to update to the newer version.

Reply:
i didn't mean for now, but in the future.. in the docs they recommend going through from warning -> error and THEN removing: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-deprecated/

i don't know how much we've been following that convention though...